_layouts/main.html

```diff
@@ -12,18 +12,22 @@
           </div>
           <div class="col-md-8">
             <p class="lead">
-The HEP Software Foundation (HSF) facilitates coordination and common efforts in high energy physics (HEP) software and computing internationally.
-           </p>
+The HEP Software Foundation facilitates cooperation and 
+<a href="{{site.baseurl}}/activities/cwp.html">common efforts</a> 
+in High Energy Physics software and computing internationally.
+</p>
       </div>
       </div>
 
 <hr>
 
 <!-- Important announcements from announcements/_posts -->
 <div>
-The HSF is guiding a community process to develop a consensus roadmap for HEP Software and Computing R&amp;D for the 2020s.
-More information can be found on the
-<a href="{{site.baseurl}}/activities/cwp.html">Community White Paper (CWP)</a> page on the HSF site.
+<p class="lead">
+The HSF has now submitted the 
+<a href="https://arxiv.org/abs/1712.06982">final roadmap paper</a> from the
```

Comment:
I suggest adding a target to open the arXiv site in a separate tab, as in other pages.

Reply:
Thanks for spotting that - done!